sourcetools/objectmodel/com/ibm/j9tools/om/io/AbstractParser.java

```diff
@@ -85,7 +85,7 @@ public AbstractParser() {
 			// Create a parser factory and configure it to enable namespaces/validation.
 			SAXParserFactory saxFactory = SAXParserFactory.newInstance();
 			saxFactory.setNamespaceAware(true);
-			saxFactory.setValidating(true);
+			saxFactory.setValidating(false);
```

Comment:
There were issues with processing some of the dtd files. I will check again now we are processing with later openj9 and jdk11 but think this may still be an issue. Will try and determine why this is failing and see whether we can resolve it that way rather than just not validating.

Reply:
Seem that having this set to 'true' works with the latest code. 
Will make another change and push the changes into this PR.

Reply:
Glad to hear that.